lib/dash.coffee

```diff
@@ -86,7 +86,10 @@ plugin = module.exports =
     return keys.map(encodeURIComponent).join(',') if keys.length
 
   getDashURI: (string, path, language, background) ->
-    link = 'dash-plugin://query=' + encodeURIComponent(string)
+    if process.env.XDG_SESSION_DESKTOP == "KDE"
+      link = 'dash-plugin:query=' + encodeURIComponent(string) #plays nice with KDE Plasma 5
+    else link = 'dash-plugin//:query=' + encodeURIComponent(string)
```

Comment:
I wouldn't rush on this one. I have no idea why KDE doesn't like host-less
URIs, so I'd like to double check that, and perhaps talk to KDE people.
On 20 Jan 2016 4:27 pm, "Blake Embrey" <notifications@github.com> wrote:

> In lib/dash.coffee
> <https://github.com/blakeembrey/atom-dash/pull/32#discussion_r50321834>:
>
> > @@ -86,7 +86,10 @@ plugin = module.exports =
> >      return keys.map(encodeURIComponent).join(',') if keys.length
> >
> >    getDashURI: (string, path, language, background) ->
> > -    link = 'dash-plugin://query=' + encodeURIComponent(string)
> > +    if process.env.XDG_SESSION_DESKTOP == "KDE"
> > +      link = 'dash-plugin:query=' + encodeURIComponent(string) #plays nice with KDE Plasma 5
> > +    else link = 'dash-plugin//:query=' + encodeURIComponent(string)
>
> If you can place it on a new line too, that would be good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blakeembrey/atom-dash/pull/32/files#r50321834>.
>